app/src/main/res/values/styles.xml

```diff
@@ -43,6 +43,17 @@
         <item name="android:lineSpacingExtra">@dimen/extra_small_lse</item>
     </style>
 
+    <style name="SearchEditText">
+        <item name="android:textColorHint">@color/divider</item>
```

Comment:
why do you use color created for a divider for hint? we already have color for hint

Reply:
fix